### PR TITLE
Adjust preemption event message to do not include preemptor pod metadata

### PR DESCRIPTION
**Pull request by mimowo**
#### What type of PR is this?

/kind bug

#### Which issue(s) this PR fixes:

Part of #114882 (in event)

#### Does this PR introduce a user-facing change?

```release-note
Fix regression in 1.23: Do not include preemptor pod metadata in the event message
```
#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/framework/preemption/preemption.go [hunk 1]
```diff
@@ -378,8 +378,8 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
-		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by %v/%v on node %v",
-			pod.Namespace, pod.Name, c.Name())
+		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by a pod scheduled by %s on node %v",
+			pod.Namespace, pod.Name, pod.Spec.SchedulerName, c.Name())
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/preemption/preemption.go:382]: pod's namespace/name should be removed:

```suggestion
			pod.Spec.SchedulerName, c.Name())
```

Comment (mimowo) [reply pkg/scheduler/framework/preemption/preemption.go]: That's right, fix pushed

**Comment by Huang-Wei** [comment]
/lgtm
/approve

Thanks.

File: pkg/scheduler/framework/preemption/preemption.go [hunk 2]
```diff
@@ -378,8 +378,8 @@ func (ev *Evaluator) prepareCandidate(ctx context.Context, c Candidate, pod *v1.
 				return
 			}
 		}
-		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by %v/%v on node %v",
-			pod.Namespace, pod.Name, c.Name())
+		fh.EventRecorder().Eventf(victim, pod, v1.EventTypeNormal, "Preempted", "Preempting", "Preempted by a pod scheduled by %s on node %v",
```
Comment (alculquicondor) [inline pkg/scheduler/framework/preemption/preemption.go:381]: -1

The scheduler name is already part of the Event metadata:


```
  Type    Reason     Age   From                                   Message
  ----    ------     ----  ----                                   -------
  Normal  Scheduled  24s   gke.io/optimize-utilization-scheduler  Successfully assigned alpha/sample-job-9fq6c-khbf4 to gke-demo-default-pool-938ff563-473d
```

Comment (Huang-Wei) [reply pkg/scheduler/framework/preemption/preemption.go:381]: Good point.

```
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  19s   default-scheduler  Successfully assigned default/pause1 to node-1
  Normal  Preempted  4s    default-scheduler  Preempted by a pod scheduled by default-scheduler on node node-1
```

@mimowo it's good to remove the duplicated scheduler-name in event, and then update the cherry-pick PRs.

Comment (mimowo) [reply pkg/scheduler/framework/preemption/preemption.go:381]: Fixed, updated the cherry-pick PR by cherry-picking the fix, PTAL.

**Pull request merged**